ansible/configs/ans-tower-prod/pre_software.yml

```diff
@@ -0,0 +1,61 @@
+---
+- name: Step 003 - Create env key
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  become: false
+  tags:
+    - step003
+    - generate_env_keys
+  tasks:
+    - name: Generate SSH keys
+      shell: ssh-keygen -b 2048 -t rsa -f "{{output_dir}}/{{env_authorized_key}}" -q -N ""
+      args:
+        creates: "{{output_dir}}/{{env_authorized_key}}"
+      when: set_env_authorized_key
+
+
+- name: Configure all hosts with Repositories, Common Files and Set environment key
+  hosts:
+    - all:!windows
+  become: true
+  gather_facts: False
+  tags:
+    - step004
+    - common_tasks
+  roles:
+    - { role: "set-repositories", when: 'repo_method is defined' }
+    - { role: "common", when: 'install_common' }
+    - { role: "set_env_authorized_key", when: 'set_env_authorized_key' }
+    - { role: "bastion-opentlc-ipa",  when: 'install_ipa_client' }
```

Comment:
shoulldn't bastion-opentlc-ipa role be called on 'bastion' and not 'all' ?

Reply:
Changed pre_software